src/store/addBook/reducer.js

```diff
@@ -0,0 +1,26 @@
+import {
+  FETCH_AUTHOR_SUCCESS,
```

Comment:
please remove google books fetching related functions and cases from the reducer

Reply:
fixed